common/table.js

```diff
@@ -132,12 +133,15 @@
                 if (!setSearchStates(scope, isBackground, searchTerm)) return;
 
                 scope.vm.page = page;
+                // TODO: Wrap catcher fn around DataUtils.getRowValuesFromPage
```

Comment:
This isn't necessary in this place. The function will be wrapped in the utils factory.